### DISCUSSION TITLE
Merging upstream changes

### DISCUSSION THREAD

**Discussion by Micrufun** [mediacms-io/mediacms#531] [Q&A] [answered]
We have merged the latest upstream changes of https://github.com/mediacms-io/mediacms/pull/519 simply by `git merge` into our fork. Then, our server works fine by just running this command:

```bash
/usr/local/bin/docker-compose restart web celery_worker celery_beat
```
# Question

We remember that there was a discussion about the docker images being updated too. Even though we didn't pull the latest docker images, our server is just fine. Now, our question is: should we be concerned about pulling the latest docker images?

**Comment by mgogoulos** [comment, marked as answer]
Hi, I don't think there should be an issue, since the new docker image is functional, and it is using the latest versions of python libraries as of requirements.txt (as far as I remember there haven't been any changes recently). 
Of course it is always a good practice if you can test it somewhere (ideally a staging environment) before this is done in a production environment!
